exercises/custom-set/custom_set.py

```diff
@@ -0,0 +1,30 @@
+class CustomSet(object):
+    def __init__(self, elements=[]):
+        pass
+
+    def empty(self):
+        pass
+
+    def __iter__(self):
+        pass
+
+    def subset(self, other):
+        pass
+
+    def disjoint(self, other):
```

Comment:
I think these should be `isdisjoint` and `issubset` to match with Python's implementation of `set`. I would also then change `empty` to `isempty` for consistency.

Reply:
Agreed.